pkgs/development/compilers/elm/packages/node/elm-pages/default.nix

```diff
@@ -1,9 +1,29 @@
 { nodePkgs, pkgs, lib, makeWrapper }:
 
-nodePkgs."elm-pages".overrideAttrs (
+let
+  ESBUILD_BINARY_PATH = lib.getExe (
+      pkgs.esbuild.override {
+        buildGoModule = args: pkgs.buildGoModule (args // rec {
+          version = "0.21.5";
```

Comment:
so I see we even need 2 different version of esbuild. I guess there was an issue with version of ES build when you tried to share the same one?

Reply:
correct